<launch>
    <node name="VLMaps_inference_node" pkg="vlmaps_ros" type="inference_node.py" output="screen"></node>
    <group ns="vlmaps_brain">
        <rosparam file="$(find vlmaps_ros)/config/vlmaps_brain.yaml" />
    </group>
</launch>